pkgs/by-name/rs/rsshub/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  fetchFromGitHub,
+  makeBinaryWrapper,
+  nodejs,
+  pnpm,
+  stdenv,
+  substitute,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "rsshub";
+  version = "0-unstable-2024-07-08";
+
+  src = fetchFromGitHub {
+    owner = "DIYgod";
+    repo = "RSSHub";
+    rev = "6a29ca395191e745f991b9a0643a2fa9a66c8730";
+    hash = "sha256-K7qNPF1vdqhNEjWcysnj20Qaltu3rnhi1bVnIEOTiuk=";
+  };
+
+  patches = [
+    (substitute {
```

Comment:
Use `substituteAll` like

https://github.com/NixOS/nixpkgs/blob/feb2849fdeb70028c70d73b848214b00d324a497/pkgs/by-name/li/librum/package.nix#L27-L31

This seems not in documentation, I'll update it later

Reply:
Hi, I used `substitute` because of https://github.com/NixOS/nixpkgs/issues/237216.